ui/src/app/views/concept-table/component.html

```diff
@@ -0,0 +1,15 @@
+<clr-datagrid [(clrDgSelected)]="selectedConcepts" [clrDgLoading]="loading">
+  <clr-dg-column>Name</clr-dg-column>
+  <clr-dg-column>Synonyms</clr-dg-column>
+  <clr-dg-column>Code</clr-dg-column>
+  <clr-dg-column>Vocabulary</clr-dg-column>
+  <clr-dg-column>Count</clr-dg-column>
+  <div class="break"></div>
+  <clr-dg-row *ngFor="let concept of concepts" [clrDgItem]="concept">
+    <clr-dg-cell>{{concept.conceptName}}</clr-dg-cell>
+    <clr-dg-cell>{{concept.conceptSynonyms.join(', ')}}</clr-dg-cell>
```

Comment:
This looks interesting in that we're assuming a particular form of concept synonym, i.e. they won't end in commas or have commas as part of the synonym. I think it's fine for this PR, but we should be conscious that synonym format might be problematic and we might need a better way to join them.

Reply:
👍